pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -537,6 +523,36 @@ func (k *Kubernetes) Transform(komposeObject kobject.KomposeObject, opt kobject.
 		service := komposeObject.ServiceConfigs[name]
 		var objects []runtime.Object
 
+		// Must build the images before conversion (got to add service.Image in case 'image' key isn't provided
+		// Check that --build is set to true
+		// Check to see if there is an InputFile (required!) before we build the container
+		// Check that there's actually a Build key
+		if opt.Build && opt.InputFiles != nil && service.Build != "" {
+
+			// Get the directory where the compose file is
+			composeFileDir, err := transformer.GetComposeFileDir(opt.InputFiles)
+			if err != nil {
+				return nil, err
+			}
+
+			// Build the container!
+			err = transformer.BuildDockerImage(service, name, composeFileDir)
+			if err != nil {
+				log.Fatalf("Unable to build Docker container for service %v: %v", name, err)
```

Comment:
@cdrage should we wrap the error (`errors.Wrap`) and send it up the call stack instead of `log.Fatalf`?